js/ad_manager_vast.js

```diff
@@ -1549,6 +1549,31 @@ OO.Ads.manager(() => {
       _handleTrackingUrls(amcAd, ['resume']);
     };
 
+    /**
+     * When the Ad Manager Controller needs to hide the overlay it will call this function. We will store the current ad
+     * for reference. Vast ad doesn't need to do much other then save the reference.
+     * @public
+     * @method Vast#hideOverlay
+     * @param {object} currentAd In order to not lose reference to the overlay object that is currently being shown, it
+     * is stored in this object
+     */
+    this.hideOverlay = (currentAd) => {
+      this.lastOverlayAd = currentAd;
```

Comment:
`this.lastOverlayAd` was removed too. Revert it too and check that you didn't miss anything else